RiotSwiftUI/Modules/Room/Composer/View/Composer.swift

```diff
@@ -140,11 +140,12 @@ struct Composer: View {
             }
             HStack(alignment: shouldFixRoundCorner ? .top : .center, spacing: 0) {
                 WysiwygComposerView(
-                    focused: $viewModel.focused,
-                    viewModel: wysiwygViewModel
+                    placeholder: viewModel.viewState.placeholder ?? "",
```

Comment:
Apparently now we don't use `$viewModel.focused`.
Shouldn't this break stuff?

Reply:
You're right, it was broken.
It should be fine now. Focus, resize and composer height seem to be OK. Let me know if it works for you.